src/components/molecules/CategoryEdit/index.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <div class="category-edit">
+    <div class="category-edit__columns">
+      <section class="category-edit-editor">
+        <app-heading :level="1">カテゴリーの更新</app-heading>
+        <div class="category-edit__back">
+          <app-router-link
+            block
+            underline
+            key-color
+            hover-opacity
+            to="/categories"
+          >
+            カテゴリー一覧へ戻る
+          </app-router-link>
+        </div>
+        <div class="category-edit-form">
+          <app-input
+            v-validate="'required'"
+            name="title"
+            type="text"
+            placeholder="カテゴリーのタイトルを入力してください。"
+            white-bg
+            data-vv-as="カテゴリーのタイトル"
+            :value="categoryTitle"
+            :error-messages="errors.collect('title')"
+            @update-value="$emit('edited-title', $event)"
+          />
+        </div>
+        <app-button
+          class="category-edit-submit"
+          button-type="submit"
+          round
+          :disabled="!disabled"
```

Comment:
[imo]

disabled じゃない時（`!disabled`）に disabled prop（`:disabled`）を true にするのは個人的に違和感があります。
「〇〇じゃない時」というふうに表現したければ別の computed 名を使用するか、以下のようにするべきかなと思います。

```suggestion
          :disabled="disabled"
```

Reply:
ありがとうございます。
disabledはアプリケーション自体で元から定義されているようで、私の方で変更することが難しそうなのですが変更した方がよろしいのでしょうか…！

Reply:
なるほど、、！
修正して OK です！
こういった違和感を今後もし持ったら相談の上、どんどん修正していってもらって大丈夫です！

Reply:
承知いたしました。「:disabled="disabled"」となるように変更いたしました！[bcc509f](https://github.com/gizumo-education/gizumo_wiki_lesson/pull/115/commits/bcc509f14dcb4f47302e22b0b78317e49eae9db4)